src/components/Drawer/ScheduleMeetingDrawer/ScheduleMeetingModal/ScheduleMeetingDrawer.tsx

```diff
@@ -0,0 +1,166 @@
+import { Drawer, Steps } from "antd";
+import React, { useEffect, useState } from "react";
+import { MeetingType } from "src/typings/Common";
+import { Inmate } from "src/typings/Inmate";
+import { useAppDispatch } from "src/redux";
+import { ComputedMeetingSlot } from "src/typings/MeetingSlot";
+import { Connection } from "src/typings/Connection";
+import { formatDate, meetingTypeToLabel } from "src/utils";
+import ScheduleMeetingTimeSlots from "./ScheduleMeetingTimeSlots";
+import ScheduleMeetingSelectType from "./ScheduleMeetingSelectType";
+import ScheduleMeetingGuestsStep from "./ScheduleMeetingGuestsStep";
+import { scheduleMeeting } from "src/redux/modules/meeting";
+import { unwrapResult } from "@reduxjs/toolkit";
+import ScheduleMeetingSuccess from "./ScheduleMeetingSuccess";
+import { format } from "date-fns";
+
+interface Props {
+  visible?: boolean;
+  onClose: () => void;
+  loadMeetingSlots: (type: MeetingType) => Promise<ComputedMeetingSlot[]>;
+  inmate: Inmate;
+  connections: Connection[];
+}
+
+enum ScheduleStep {
+  MEETING_TYPE = 0,
+  TIME = 1,
+  GUESTS = 2,
+  SUCCESS = 4,
```

Comment:
if the exact values are unimportant, you can omit declaring them. it helps discourage people from depending on them:

```suggestion
  MEETING_TYPE,
  TIME,
  GUESTS,
  SUCCESS,
```